jsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Looks like the vscode configuration file to get javascript intellisense, not sure it should be there. Was it committed on purpose?